### ISSUE TITLE
Installation on unix systems

### ISSUE THREAD

**Issue by Ri0n** [azadkuh/qhttp#19]
It was surprising generated makefiles does not install library/headers into system. Instead it creates some xbin and install there.

Please make proper installation
1. Executables into /usr/bin by default
2. Libraries into /usr/lib by default
3. Headers into /usr/include

Where /usr prefix may be redefined and full path to lib directory may be redefined as well.

So project file should use PREFIX and LIBDIR variables passed to qmake and use some default values is not passed.

**Comment by azadkuh** [comment]
yes it would be nice to have a more generalized `*.pro`/`*.pri` combination. 
(`qhttp` is mostly used as an static lib by myself)

**Comment by giraldeau** [comment]
I faced the same issue and I finally used the solution suggested by @azadkuh. I added `CONFIG += staticlib` in src/src.pro and then xbin/libqhttp.a is generated in addition to the shared library. To link, use -l:libqhttp.a instead of -lqhttp.
